paragraph/session.py

```diff
@@ -11,10 +12,10 @@
 
 @contextmanager
 def eager_mode():
```

Comment:
Could add some documentation. I have only a rough idea how to use this one.

Reply:
Will do.